mllib/src/main/scala/org/apache/spark/mllib/linalg/BLAS.scala

```diff
@@ -235,12 +235,23 @@ private[spark] object BLAS extends Serializable with Logging {
    * @param x the vector x that contains the n elements.
    * @param A the symmetric matrix A. Size of n x n.
    */
-  def syr(alpha: Double, x: DenseVector, A: DenseMatrix) {
+
+  // Wrapper around the dense and sparse methods.
```

Comment:
This is not part of the JavaDoc. We can either move it to the JavaDoc or to the method closure.